kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -99,6 +103,7 @@
     data() {
       return {
         bannerOpened: false,
+        isDeviceRoute: window.location.href.includes('device'),
```

Comment:
This could result in a user not seeing the banner in Learn when they should [as we use a `deviceId` path ](https://github.com/learningequality/kolibri/blob/35461b94b55ff79dddc250e50b40a191eb0fab01/kolibri/plugins/learn/assets/src/routes/index.js#L28)in the URL when handling remote resource navigation in the Library. 

So in those cases the banner would not show when it ought to.

I think that you can instead use `window.location.pathname.includes('device')` here instead as that will only return the language code & plugin name (ie, `/en/learn` or `/ar/device`)

Reply:
Looking at the issue, it seems like we only want this to appear on the Learn page? Maybe we should just remove the banner entirely from the AppBarPage, and instead put it into the relevant place in the learn page instead? That solves any of this second guessing of the URL. Remove from here: https://github.com/learningequality/kolibri/blob/35461b94b55ff79dddc250e50b40a191eb0fab01/kolibri/core/assets/src/views/CorePage/AppBarPage.vue#L26

Instead add it here, with a `v-if="page === 'AppBarPage'"`?

Reply:
Hi @iskipu - thanks for your contribution here. After chatting with @rtibbles further and reviewing the underlying code again, my guidance in the issue was definitely incorrect.  I'm sorry about that! I wrote the issue too quickly and didn't read through the code closely enough before I gave a suggested approach.

> Because the StorageNotificationBanner is widely used and exists in core, this should not be moved (i.e. moving the component itself only to Learn), but rather, the condition should be updated.

This was not the right direction. I reviewed the spec, and the code, and this `<StorageNotification/>` component does only need to be used in the `Learn` plugin. So, the exact opposite of what I wrote here -- what @rtibbles said above -- is the way to go. I've updated the issue description to match. Apologies, and thanks again for your interest in contributing.

Reply:
> what @rtibbles said above -- is the way to go. 

Hi @marcellamaki no worries, i have made a new commit i hope this is what you are expecting, let me know if i did something wrong.